src/Entities/Player.h

```diff
@@ -228,8 +228,8 @@ class cPlayer :
 
 	// tolua_begin
 
-	/** Opens the specified window; closes the current one first using CloseWindow() */
-	void OpenWindow(cWindow * a_Window);
+	/** Opens the specified window; closes the current one first using CloseWindow(). This is exported by tolua and it adds the nil check we want */
```

Comment:
I don't think the extra comment is necessary. It's something you've just fixed and no-one will be interested in it anymore. Export to Lua is visible because of the `tolua` comment just above.